web/hooks/use-bets.ts

```diff
@@ -37,9 +29,12 @@ export const useBetsWithoutAntes = (contract: Contract, initialBets: Bet[]) => {
     withoutAnteBets(contract, initialBets)
   )
   useEffect(() => {
-    return listenForBets(contract.id, (bets) => {
-      setBets(withoutAnteBets(contract, bets).sort((b) => b.createdTime))
-    })
+    return listenForBets(
+      (bets) => {
+        setBets(withoutAnteBets(contract, bets).sort((b) => b.createdTime))
+      },
+      { contractId: contract.id }
```

Comment:
I think you can add the `filterAntes: true` filter and then delete the `withoutAnteBets` function. I believe it's from a time long ago when ante bets used to be just the first two bets.

Reply:
OK, I wasn't confident about what the deal was with that first two bets business. If you think it's OK I will happily clean this up.